test/src/test/java/jenkins/tasks/SimpleBuildWrapperTest.java

```diff
@@ -70,6 +84,61 @@
         }
     }
 
+    @Test public void envOverrideExpand() throws Exception {
+        Assume.assumeFalse(Functions.isWindows());
```

Comment:
Could do so using XShell plugin. `CaptureEnvironmentBuilder` works on anything, but I wanted to check variables actually passed to a `Launcher` too.